src/main/java/com/exasol/dbbuilder/User.java

```diff
@@ -118,6 +118,7 @@ public User grant(final DatabaseObject object, final ObjectPrivilege... privileg
      * @param privileges system privileges
      * @return {@link User} instance for fluent programming
      */
+    // [dsn~granting-system-privileges-to-database-users~1]
```

Comment:
```suggestion
    // [impl->dsn~granting-system-privileges-to-database-users~1]
```

Please run OFT to make sure tracing is done correctly. Latest version can be downloaded here:
https://search.maven.org/remotecontent?filepath=org/itsallcode/openfasttrace/openfasttrace/3.0.2/openfasttrace-3.0.2.jar